Automattic/wp-calypso
# Products: Move Jetpack product price matrix to constants

#### Changes proposed in this Pull Request

* Move the Jetpack product price matrix to the product values constants.

#### Testing instructions

* Checkout this branch.
* Go to http://calypso.localhost:3000/plans/:site where `:site` is a Jetpack site.
* Verify things still work like they do on `wpcalypso.wordpress.com`.
* Go to http://calypso.localhost:3000/devdocs/blocks/product-selector
* Verify things still work like they do on `wpcalypso.wordpress.com`.

Fixes no known issue, but addresses feedback from https://github.com/Automattic/wp-calypso/pull/37023#discussion_r338604777 by @delawski
